specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -2443,6 +2620,262 @@
+        },
+        "temperature": {
+          "$ref": "#/definitions/WeatherUnitYearMaxMinAvg",
+          "description": "Temperature value."
+        },
+        "precipitation": {
+          "$ref": "#/definitions/WeatherUnitYearMax",
+          "description": "Maximum amount of precipitation (liquid equivalent) that has fallen."
+        },
+        "snow": {
+          "$ref": "#/definitions/WeatherUnitYearMax",
+          "description": "Maximum snow."
+        }
+      }
+    },
+    "DailyHistoricalNormalsResponse": {
+      "type": "object",
+      "properties": {
+        "historicalNormals": {
+          "type": "array",
+          "description": "Historical normals for each requested day.",
+          "items": {
+            "$ref": "#/definitions/DailyHistoricalNormals"
+          }
+        }
+      }
+    },
+    "DailyHistoricalNormals": {
+      "type": "object",
+      "properties": {
+        "date": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Date and time of the current observation displayed in ISO 8601 format, for example, 2019-10-27T19:39:57-08:00."
+        },
+        "temperature": {
+          "$ref": "#/definitions/WeatherUnitMaxMinAvg",
+          "description": "Temperature values."
+        },
+        "degreeDaySummary": {
+          "$ref": "#/definitions/DegreeDaySummary",
+          "description": "Summary of heating or cooling degree day information"
+        },
+        "precipitation": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "The amount of precipitation (liquid equivalent) that has fallen."
+        }
+      }
+    },
+    "HistoricalSummaryResponse": {
+      "type": "object",
+      "properties": {
+        "historicalActuals": {
+          "$ref": "#/definitions/MonthlyHistoricalActuals",
+          "description": "Historical actuals for the requested month."
+        },
+        "historicalRecords": {
+          "$ref": "#/definitions/MonthlyHistoricalRecords",
+          "description": "Historical records for the requested month."
+        },
+        "historicalNormals": {
+          "$ref": "#/definitions/MonthlyHistoricalNormals",
+          "description": "Historical normals for the requested month."
+        }
+      }
+    },
+    "MonthlyHistoricalActuals": {
+      "type": "object",
+      "properties": {
+        "temperature": {
+          "$ref": "#/definitions/WeatherUnitMaxMinAvg",
+          "description": "Temperature value."
+        },
+        "degreeDaySummary": {
+          "$ref": "#/definitions/DegreeDaySummary",
+          "description": "Summary of Heating or Cooling Degree Day information."
+        },
+        "precipitation": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "The amount of precipitation (liquid equivalent) that has fallen."
+        },
+        "snow": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Snow."
```

Comment:
With the change from 'Snow' to Snowfall', please also change the description for Snowfall to be:

The amount of snow that has fallen.